src/App.js

```diff
@@ -1,5 +1,27 @@
+import React, { useState } from 'react';
+// import { Route, Routes } from 'react-router-dom';
+import Navbar from './layout/Navbar';
+
 function App() {
-  return <div>Welcome to Movies App</div>;
+  const [classValue, setClassValue] = useState('show');
+  // const [showNav, setShowNav] = useState('block');
+  const toggleMenu = () => {
+    if (classValue === 'show') {
+      setClassValue('hide');
+      // setShowNav('none');
+    } else {
+      setClassValue('show');
+      // setShowNav('block');
+    }
+  };
+  return (
+    <div>
+      <Navbar
+        classValue={classValue}
+        toggleMenu={toggleMenu}
+      />
+    </div>
+  );
```

Comment:
this is an optional suggestion to make your code more readable... You may include these to navbar component and here you may include <Navbar /> component only